python/common/org/python/stdlib/datetime/TimeDelta.java

```diff
@@ -60,28 +61,65 @@ public TimeDelta(org.python.Object[] args, java.util.Map<java.lang.String, org.p
 	    }
 	}
 
-	if (args.length == 3) {
-	    this.days = args[0];
-	    this.seconds = args[1];
-	    this.microseconds = args[2];
-	} else if (args.length == 2) {
-	    this.days = args[0];
-	    this.seconds = args[1];
-	    this.microseconds = org.python.types.Int.getInt(0);
-
-	} else if (args.length == 1) {
-	    this.days = args[0];
-	    this.seconds = org.python.types.Int.getInt(0);
-	    this.microseconds = org.python.types.Int.getInt(0);
-
+	if (args.length == 7) {
```

Comment:
Stylecheck complains:
'}' at column 9 should be on the same line as
the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else or try/catch/finally). [RightCurly]

It should be
`} else if () {`
not
`}`
`else if () {`